utils.py

```diff
@@ -178,8 +178,10 @@ def write_images(
     image_grid = image_grid.permute(1, 2, 0).cpu().detach().numpy()
 
     if comet_exp is not None:
-        comet_exp.log_image(image_grid, name="test_iter_" + str(curr_iter))
-
+        if is_test:
+            comet_exp.log_image(image_grid, name="test_iter_" + str(curr_iter))
+        else: 
+            comet_exp.log_image(image_grid, name="train_iter_" + str(curr_iter))
```

Comment:
you should always specify `step=` in the comet logs ; I know it works fine without it but sometimes it's messed up for some unknown reason, where you'd see thousands of steps with missing data. And this prevents that

Reply:
I fixed that